straxen/plugins/events/event_pattern_fit.py

```diff
@@ -13,6 +13,7 @@ class EventPatternFit(strax.Plugin):
     """Plugin that provides patter information for events."""
 
     depends_on = ("event_area_per_channel", "event_basics", "event_positions")
+    save_when = strax.SaveWhen.ALWAYS
```

Comment:
This line is not needed because the default `save_when` is `ALWAYS`.

Reply:
Meanwhile, you should make `event_area_per_channel` always saved, but in another plugin.